app/views/spaces/show.html.erb

```diff
@@ -2,14 +2,13 @@
 <p>Find me in app/views/spaces/show.html.erb</p>
 
 <ul class="list-group">
-  <% spaces.each do |space| %>
     <li class="list-group-item">
-      <%= space.location %>
-      <%= space.description %>
-      <%= space.volume %>
-      <%= space.price_per_month %>
+      <%= @space.location %>
+      <%= @space.description %>
+      <%= @space.volume %>
+      <%= @space.price_per_month %>
       </li>
```

Comment:
I am just assuming that you mean to have a list of the details like so:

- Location
- Description
- Volume
- Price per month

For this, you would have to put the `<li>` tags around each list item.
```
<li> Location </li>
<li> Description </li>
...
```